.github/workflows/scheduleCurrentLinkCheck.yaml

```diff
@@ -0,0 +1,20 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Schedule Current Link Check
+
+# Controls when the workflow will run
+on:
+  # The schedule event allows you to trigger a workflow at a scheduled time.
+  schedule:
```

Comment:
let's add one https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch trigger too